python/kvikio/_lib/libnvcomp.pyx

```diff
@@ -23,23 +23,27 @@
 from enum import Enum
 
 import cupy as cp
+import cython
 
-from libc.stdint cimport uintptr_t
-from libcpp cimport bool
+from cython.operator cimport dereference
+from libc.stdint cimport int32_t, uint8_t, uintptr_t
+from libcpp cimport bool, nullptr
+from libcpp.memory cimport make_shared, shared_ptr
+from libcpp.utility cimport move
 
 from kvikio._lib.arr cimport Array
 from kvikio._lib.nvcomp_cxx_api cimport (
-    __CascadedCompressor,
-    __CascadedDecompressor,
-    __LZ4Compressor,
-    __LZ4Decompressor,
+    ANSManager,
+    CascadedManager,
+    CompressionConfig,
+    DecompressionConfig,
+    LZ4Manager,
+    SnappyManager,
```

Comment:
Just double-checking that gdeflate and bitcomp are intentionally excluded here.  I noticed that the equivalents in nvcomp.py raise NotImplementedError, but ANS does the same there, but ANS is in in this list and gdeflate and bitcomp aren't.

Reply:
Thanks @ndickson-nvidia . ANS was accidentally left in the import list after I tested to see if I "already had it". I did not. I could actually include the bindings for ANS, GDeflate, and Bitcomp in the python and it would throw library import errors if the user tried to use them without them being installed. I can't test them without having gone through all those processes yet, hence not having done so. I accidentally left ANSManager in after testing it.

Reply:
Could we provide an error such as "for 2.2.0, you need to install the closed source compressors or upgrade to nvCOMP 2.3"? (now all the compressors are included by default)

Reply:
All the compressors are in 2.3.3? Perhaps I made a mistake, I will circle around and recheck it because I didn't get the version properly updated from 2.2.0 to 2.3.3 until yesterday. I'd love to include all the compressors.

Reply:
Yes. When we moved to closed source we removed the need to install separate packages for some of the compressors.